core/src/main/java/org/cru/contentscoring/core/servlets/SyncScoreServlet.java

```diff
@@ -73,7 +73,8 @@ protected void doPost(
                     .setHost(uri.getHost())
                     .build()
                     .toString();
-                pathFinderResponse = client.target(serverUri + "/bin/cru/path/finder")
+                LOG.debug("Calling {} with path {}", serverUri + "/bin/cru/path/finder.html", incomingUri);
+                pathFinderResponse = client.target(serverUri + "/bin/cru/path/finder.html")
```

Comment:
I think it's a little weird to be returning a text blob from a .html url. Did you look into how hard it'd be to use a different extension, perhaps `.../finder.txt`?

Reply:
Should be easy. Originally I was doing .html because the url was already coming in with it in many cases, but I ended up having to tack it on here manually anyway so it shouldn't be any harder to do a different extension.

Reply:
cool